### PR TITLE
Documentation: Update installation documentation

### PR DESCRIPTION
**Pull request by chancez**
- Adds a new Openshift install doc on using openshift-ansible for installing into Openshift.
- Renames the dev install doc to manual-install
- Moves the previous main install documentation into it's own olm-install document
- From the main install documentation, link to each of these individual install docs based on platform and version requirements.

### CODE REVIEW

File: Documentation/ansible-install.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+openshift_metering_install: true
+```
+
+Finally, run the playbook:
+
+```
+ansible-playbook playbooks/openshift-metering/config.yml
+```
+
+If your on GCP use the following instead:
```
Comment (tschuy) [inline Documentation/ansible-install.md]: `If you're on GCP`

File: Documentation/ansible-install.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Installation using openshift-ansible
+
+Using ansible is the recommend installation method for installing Metering on to an Openshift cluster.
+
+Within the [openshift-ansible repo][openshift-ansible] is where the [openshift-metering playbook][metering-playbook] is located.
```
Comment (tschuy) [inline Documentation/ansible-install.md]: either

`Within the [openshift-ansible repo][openshift-ansible] is the [openshift-metering playbook][metering-playbook].`

or 

`The [openshift-metering playbook][metering-playbook] is located within the [openshift-ansible repo][openshift-ansible].`

Comment (chancez) [reply Documentation/ansible-install.md]: Yeah wow the existing phrasing was pretty bad. Thanks.

File: Documentation/ansible-install.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Finally, run the playbook:
+
+```
+ansible-playbook playbooks/openshift-metering/config.yml
+```
+
+If you're on GCP use the following instead:
+
+```bash
+ansible-playbook playbooks/openshift-metering/gcp-config.yml
```
Comment (pruan-rht) [inline Documentation/ansible-install.md]: @chancez, would be helpful for newbies like me to list out the expected pods to be found after a successful installation.

**Pull request merged**